### ISSUE TITLE
Host on Packagist?

### ISSUE THREAD

**Issue by nick-andren** [paslandau/guzzle-auto-charset-encoding-subscriber#1]
Hey,

I noticed that this repo is hosted on packages.myseosolution.de. I was wondering, is there a reason why it and its related packages aren't hosted on Packagist? Is there any way that I could help with this?

**Comment by paslandau** [comment]
Hey nick,

there are three reasons:
1. I actually love packagist but I didn't feel the code quality was high enough to let others rely on it yet
2. It's actually [quite hard to delete stuff on packagist](https://github.com/composer/packagist/issues/166) which (considering 1.) isn't something I want right now
3. the current setup makes just a little difference in terms of installation effort for the end user

Orginially, I wanted the code to be github-only (see the ["Installation" instruction on v.0.1](https://github.com/paslandau/guzzle-auto-charset-encoding-subscriber/tree/0.1.0#installation) for instance) but I've got other some inter-dependencies with other packages of mine (e.g. [web-utility](https://github.com/paslandau/web-utility) - a package that isn't even documented yet) and composer can't resolve second level depencies if they are not hosted on Packagist. So I went for a local satis setup on http://myseosolution.de - hence the current installation instructions.

In the end (hopefully within the next few months), I intend to host the package on packagist but I'd like to get some housekeeping (code-wise) done until I do so.

Cheers
Pascal
